ci-operator/step-registry/baremetal/lab/kdump/baremetal-lab-kdump-commands.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+set -o errtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+if [ "${ENABLE_KGUMP:-false}" != "true" ]; then
+  echo "kernel dump is not enabled. Skipping..."
+  exit 0
+fi
+
+# Trap to kill children processes
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM ERR
+# Save exit code for must-gather to generate junit
+trap 'echo "$?" > "${SHARED_DIR}/install-status.txt"' TERM ERR
+
+[ -z "${AUX_HOST}" ] && { echo "AUX_HOST is not filled. Failing."; exit 1; }
+[ -z "${architecture}" ] && { echo "\$architecture is not filled. Failing."; exit 1; }
+
+workdir=`mktemp -d`
+
+cp ${CLUSTER_PROFILE_DIR}/pull-secret /tmp/pull-secret
+oc registry login --to /tmp/pull-secret
+ocp_version=$(oc adm release info --registry-config /tmp/pull-secret ${RELEASE_IMAGE_LATEST} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
```

Comment:
Why do you need to copy secret to `tmp` directory? cant we directly use `${CLUSTER_PROFILE_DIR}/pull-secret`?
```suggestion
ocp_version=$(oc adm release info --registry-config ${CLUSTER_PROFILE_DIR}/pull-secret ${RELEASE_IMAGE_LATEST} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
```